pybliometrics/utils/get_content.py

```diff
@@ -111,13 +111,14 @@ def get_content(url, api, params=None, **kwds):
     # Either use token header or key header
     if token_key:
         header = token_header
+        resp = session.get(url, headers=header, params=params, timeout=timeout)
+
     else:
         header = key_header
+        resp = session.get(url, headers=header, params=params, timeout=timeout, proxies=proxies)
```

Comment:
I don't think that's necessary. If `token_key=True`, you can simply set proxies to None or an empty dict.